Documentation/ceph-csi-troubleshooting.md

```diff
@@ -441,3 +441,79 @@ $ rbd ls --id=csi-rbd-node -m=10.111.136.166:6789 --key=AQDpIQhg+v83EhAAgLboWIbl
 ```
 
 Where `-m` is one of the mon endpoints and the `--key` is the key used by the CSI driver for accessing the Ceph cluster.
+
+
+## Volume `is still being used` error
+
+### Symptoms
+
+If you are using the CSI rbd plugin and you observe an error from a pod using
+Ceph for block storage that includes the error string `is still being used`.
```

Comment:
This happens for the RWO volumes and user/admin need to make sure that no application pods (running) are using the pvc.

Reply:
This is a far point.  I am planning to address these review comments later this week.

Reply:
I added a brief sentence and a yaml snippet of `ReadWriteOnce` to the symptoms section.